vehicle/raw_vehicle_cmd_converter/schema/raw_vehicle_cmd_converter.schema.json

```diff
@@ -142,9 +142,6 @@
         }
       },
       "required": [
-        "csv_path_accel_map",
-        "csv_path_brake_map",
-        "csv_path_steer_map",
```

Comment:
Instead of removing these lines, I focused on fixing the source of the problem.

- https://github.com/autowarefoundation/autoware.universe/pull/6476